build.sbt

```diff
@@ -89,6 +95,43 @@ lazy val commonSettings = Seq(
   }
 )
 
+lazy val mimaSettings = Seq(
+  resolvers += "Sonatype OSS Staging" at "https://oss.sonatype.org/content/repositories/staging",
```

Comment:
Why do we need this?

Reply:
To avoid Travis CI build failure after pushing a new tag during the release, while just released artifacts are not available on the Maven Central repo still.

Reply:
Ah, makes sense.
Thank you!